pyproject.toml

```diff
@@ -56,15 +55,12 @@ changelog = "https://www.pygmt.org/latest/changes.html"
 
 [tool.setuptools]
 platforms = ["Any"]
-include-package-data = true
+license-files = ["LICENSE.txt"]
```

Comment:
Removing the `include-package-data = true` line is the same as keeping it, since that's the default according to https://setuptools.pypa.io/en/latest/userguide/datafiles.html#include-package-data. Maybe just keep the line?

Reply:
We don't have package data to include, so keeping this line may be more confusing (although the default is still `true`).

Reply:
Mm ok, I see you removed the `[tool.setuptools.package-data]` section, so should be ok to remove it.